test/k8sT/manifests/cilium-operator-patch.yaml

```diff
@@ -0,0 +1,14 @@
+---
+metadata:
+  namespace: kube-system
+spec:
+  template:
+    spec:
+      containers:
+      - image: k8s1:5000/cilium/operator:latest
+        imagePullPolicy: Always
+        name: cilium-operator
+      volumes:
```

Comment:
Is this to accommodate the etcd-operator? Comments would help.

Reply:
Hi, 

I copy and paste from other patch here, I'm not sure why it's needed, maybe it's related to etcd-operator but I'm not sure right now. 

Regards